templates/ssl_ratios_v1.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-derived-datasets.telemetry.ssl_ratios_v1` AS
+WITH windowed AS (
+  SELECT
+    *,
+    SUM(ssl_loads) OVER w1 + SUM(non_ssl_loads) OVER w1 AS total_loads
```

Comment:
calculating `total_loads` using a window over the raw table makes sure we don't need to choose an aggregation for a constant value.